system/filesystem.py

```diff
@@ -64,6 +64,37 @@
 - filesystem: fstype=ext4 dev=/dev/sdb1 opts="-cc"
 '''
 
+def human_to_bytes(data):
```

Comment:
There is already human_to_bytes in the basic module library, any reason to not reuse it ?